packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -487,6 +488,13 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
       this.communityState = ServiceState.LAUNCHING
       await this.launchCommunity(args)
     })
+    this.socketService.on(
+      SocketActionTypes.SEND_COMMUNITY_METADATA,
+      async (payload: CommunityMetadata, callback: (response?: CommunityMetadata) => void) => {
+        const meta = await this.storageService?.updateCommunityMetadata(payload)
+        callback(meta)
+      }
+    )
```

Comment:
This demonstrates the socket.io acknowledgement feature on the backend. https://socket.io/docs/v3/emitting-events/#acknowledgements